packages/spark-core/base/_typography.scss

```diff
@@ -2,114 +2,114 @@
 // Typography
 // ==================================================================
 
-@if $using-webfont-body {
-  @font-face {
-    font-family: $body-font-name;
-    src: $body-font-src;
-    font-weight: $body-font-weight;
-  }
-  @font-face {
-    font-family: $body-font-name;
-    src: $body-bold-font-src;
-    font-weight: $body-bold-font-weight;
-  }
-}
-@if $using-webfont-heading {
-  @font-face {
-    font-family: $heading-font-name;
-    src: $heading-font-src;
-    font-weight: $heading-font-weight;
-  }
-  @font-face {
-    font-family: $heading-font-name;
-    src: $heading-bold-font-src;
-    font-weight: $heading-bold-font-weight;
-  }
-}
-@if $using-webfont-alternate {
-  @font-face {
-    font-family: $alternate-font-name;
-    src: $alternate-font-src;
-    font-weight: $alternate-font-weight;
+@if $sprk-using-webfonts {
+  @each $src, $name, $weight in $sprk-webfonts {
+    @font-face {
+      font-display: $sprk-font-display;
+      font-family: $name;
+      src: $src;
+      font-weight: $weight;
+    }
   }
 }
 
-.sprk-b-TypeDisplayOne,
-.sprk-b-TypeDisplayTwo {
-  font-family: $alternate-font-stack;
-}
-
-.sprk-b-TypeDisplayThree,
-.sprk-b-TypeDisplayFour,
-.sprk-b-TypeDisplayFive,
-.sprk-b-TypeDisplaySix,
-.sprk-b-TypeDisplaySeven,
-.sprk-b-TypeDisplayEight {
-  font-family: $heading-font-stack;
-}
-
-.sprk-b-TypeBodyOne,
-.sprk-b-TypeBodyTwo,
-.sprk-b-TypeBodyThree,
-.sprk-b-TypeBodyFour {
-  font-family: $body-font-stack;
-}
-
+h1,
 .sprk-b-TypeDisplayOne {
   @include TextCrop(
-    $line-height: $line-height-display-one,
+    $line-height: $sprk-line-height-display-one-narrow,
     $top-crop: $line-height-collision-top-crop,
     $bottom-crop: $line-height-collision-bottom-crop,
     $crop-font-size: (
-      strip-unit($font-size-display-one) * 16
+      strip-unit($sprk-font-size-display-one-narrow) * 16
     ),
-    $crop-line-height: $line-height-display-one
+    $crop-line-height: $sprk-line-height-display-one-narrow
   );
 
   color: $color-display-one;
-  font-size: $font-size-display-one;
+  font-family: $sprk-font-family-display-one;
+  font-size: $sprk-font-size-display-one-narrow;
```

Comment:
I know it sounds pedantic lol but I wanted to comment my thoughts anyway, not that you need to change this ASAP or anything. The default of `$sprk-font-size-display-one-narrow` and then `$font-size-display-one` on large screens makes it so that `$font-size-display-one` is the default by it's naming. Which, goes against the mobile-first thinking. I think the mobile-first approach would have them named as `$sprk-font-size-display-one` and then `$sprk-font-size-display-one-wide` would be the var for size at larger views. Or something like that! Again, these are just my thoughts.

Reply:
agreed, i'll update it